File: app/src/main/java/ru/evgeniy/androidacademy/news/ItemDecorator.java
package ru.evgeniy.androidacademy.news;

import android.graphics.Rect;
import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;

public class ItemDecorator extends RecyclerView.ItemDecoration {

    private int spacing;

    public ItemDecorator(int spacing) {
        this.spacing = spacing;
    }

    @Override
    public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull RecyclerView parent, @NonNull RecyclerView.State state) {
            outRect.bottom = spacing;
            outRect.top = spacing;
            outRect.left = spacing;
            outRect.right = spacing;

    }
}

File: app/src/main/java/ru/evgeniy/nytimes/screens/newsList/ItemDecorator.kt
package ru.evgeniy.nytimes.screens.newsList

import android.graphics.Rect
import android.support.v7.widget.RecyclerView
import android.view.View

class ItemDecorator(private val spacing: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        outRect.bottom = spacing
        outRect.top = spacing
        outRect.left = spacing
        outRect.right = spacing

    }
}
